### DISCUSSION TITLE
Empty screen

### DISCUSSION THREAD

**Discussion by rajanvishwa10** [filamentphp/filament#15175] [Help]
### Package

Table builder

### Package Version

v3.2.0

### How can we help you?

http://127.0.0.1:8000/admin/leadershipTeams?tableSearch=test

public static function table(Table $table): Table
    {
        return $table
            ->columns([
                Tables\Columns\TextColumn::make('member_name')->searchable(),
                Tables\Columns\TextColumn::make('designation'),
                Tables\Columns\ImageColumn::make('image')
            ])
            ->filters([
                //
            ])
            ->actions([
                Tables\Actions\EditAction::make(),
                Tables\Actions\DeleteAction::make()
                    ->requiresConfirmation(),
            ])
            ->bulkActions([
                Tables\Actions\BulkActionGroup::make([
                    Tables\Actions\DeleteBulkAction::make(),
                ]),
            ]);
    }
    
    this is my table, whenever I am trying to search something a black empty screen appears and same thing happens when I am deleting an entry because I am using requiresConfirmation
    

![image](https://github.com/user-attachments/assets/ed5335be-86d3-4c19-b4b3-ef096b9a2f11)

**Comment by vit-ndpm** [comment]
Me too struggling with same issue

**Comment by leandrocfe** [comment]
`php artisan about --only=Filament`

What is the output?

**Comment by rajanvishwa10** [comment]
Filament .........................................................................................................................................  
  Packages ......................................................................................... filament, forms, notifications, support, tables  
  Version ................................................................................................................................... v3.2.0  
  Views .............................................................................................................................. NOT PUBLISHED

**Reply by leandrocfe** [reply]
add `^` to the Filament version in the composer.json file
`"filament/filament": "^3.2",`

Run `composer update -W`
